core/src/main/java/org/infinispan/xsite/irac/DefaultIracManager.java

```diff
@@ -554,55 +548,46 @@ synchronized boolean isCompleted() {
       }
    }
 
-   @ManagedAttribute(description = "Number of keys that need to be sent to remote site(s)", displayName = "XsiteQueueSize", measurementType = MeasurementType.DYNAMIC)
+   @ManagedAttribute(description = "Number of keys that need to be sent to remote site(s)",
+         displayName = "Queue size",
+         measurementType = MeasurementType.DYNAMIC)
    public int getQueueSize() {
-      if (!getStatisticsEnabled()) {
-         return -1;
-      }
-      return updatedKeys.size();
+      return getStatisticsEnabled() ? updatedKeys.size() : -1;
    }
 
-   @ManagedAttribute(description = "Number of Conflicts", displayName = "Number of Conflicts", measurementType = MeasurementType.TRENDSUP)
+   @ManagedAttribute(description = "Number of Conflicts",
+         displayName = "Number of conflicts",
+         measurementType = MeasurementType.TRENDSUP)
    public long getNumberOfConflicts() {
-      if (!getStatisticsEnabled()) {
-         return -1;
-      }
-
-      long conflictsCounts = numberOfConflictLocalWins.longValue() + numberOfConflictRemoteWins.longValue()
-            + numberOfConflictMerged.longValue();
-      return conflictsCounts;
+      return getStatisticsEnabled() ? sumConflicts() : -1;
    }
 
-   @ManagedAttribute(description = "Number of Discards", displayName = "Number of Discards", measurementType = MeasurementType.TRENDSUP)
+   @ManagedAttribute(description = "Number of Discards",
+         displayName = "Number of discards",
+         measurementType = MeasurementType.TRENDSUP)
    public long getNumberOfDiscards() {
-      if (!getStatisticsEnabled()) {
-         return -1;
-      }
-      return discardCounts.longValue();
+      return getStatisticsEnabled() ? discardCounts.longValue() : -1;
    }
 
-   @ManagedAttribute(description = "Increases the count of conflicts if merge policy discards the remote update.", displayName = "NumberOfConflictLocalWins", measurementType = MeasurementType.TRENDSUP)
-   public long getNumberOfConflictLocalWins() {
-      if (!getStatisticsEnabled()) {
-         return -1;
-      }
-      return numberOfConflictLocalWins.longValue();
+   @ManagedAttribute(description = "Number of conflicts where the merge policy discards the remote update.",
+         displayName = "Number of conflicts where local value wins",
```

Comment:
"wins" might not be the clearest meaning. Consider "Number of conflicts where the local value is used."